core/client/app/controllers/setup/two.js

```diff
@@ -55,9 +55,14 @@ export default Ember.Controller.extend(ValidationEngine, {
                         }]
                     }
                 }).then(function () {
+                    // Don't call the success handler, otherwise we will be redirected to admin
+                    self.controllerFor('application').set('skipAuthSuccessHandler', true);
+
                     self.get('session').authenticate('simple-auth-authenticator:oauth2-password-grant', {
                         identification: self.get('email'),
                         password: self.get('password')
+                    }).then(function () {
```

Comment:
Since this controller sticks around in memory after we move off the screen we should `self.set('password', '')` after successfully authenticating so we don't hold onto a plaintext copy of the password any longer than necessary.